src/components/calculator-component.js

```diff
@@ -0,0 +1,13 @@
+function calculator(id,buttons) {
```

Comment:
you don't need to put `component` in this file name, because everything inside of this folder must be a component ;)

Reply:
ya, i removed this component buttons

Reply:
it's just with the id now